### PR TITLE
Adding contact and about pages

### PR DESCRIPTION
**Pull request by george-icarcover**
Pages: 
- /contact 
- /about

**Contact: (desktop)**
https://www.figma.com/file/f0PiJaMbj3moDGXq45iAA1/Coverland-Website-Design?type=design&node-id=392%3A24777&mode=design&t=3Bdrom3tYBs3lWtf-1

**About: (desktop)**
https://www.figma.com/file/f0PiJaMbj3moDGXq45iAA1/Coverland-Website-Design?type=design&node-id=387%3A14322&mode=design&t=3Bdrom3tYBs3lWtf-1

**Contact: (mobile)**
https://www.figma.com/file/f0PiJaMbj3moDGXq45iAA1/Coverland-Website-Design?type=design&node-id=392%3A25546&mode=design&t=3Bdrom3tYBs3lWtf-1

**About: (mobile)**
https://www.figma.com/file/f0PiJaMbj3moDGXq45iAA1/Coverland-Website-Design?type=design&node-id=392%3A22091&mode=design&t=3Bdrom3tYBs3lWtf-1

### CODE REVIEW

File: src/app/(main)/about/page.tsx [hunk 1]
```diff
@@ -1,30 +1,138 @@
...
+  fallback: ['Helvetica', 'sans-serif'],
+  variable: '--font-raleway',
+});
+const lato = Lato({
+  weight: ['100', '400', '700', '900'],
+  subsets: ['latin'],
+  display: 'swap',
+  fallback: ['Helvetica', 'sans-serif'],
+  variable: '--font-lato',
+});
```
Comment (devwilliamy) [inline src/app/(main)/about/page.tsx:25]: Why do you need this here?

File: src/app/(main)/about/page.tsx [hunk 2]
```diff
@@ -1,30 +1,138 @@
...
+          src={Banner}
+        />
+        <div className="absolute inset-0 flex items-center justify-center">
+          <p
+            className={`${raleway.className} text-[32px] font-bold leading-[32px] text-white lg:text-[40px] lg:leading-[44px]`}
+          >
+            About Coverland
+          </p>
+        </div>
+      </header>
```
Comment (devwilliamy) [inline src/app/(main)/about/page.tsx:61]: You can just use my Header component from the policy pages, might have to spruce it up a bit but you should be able to reuse it

**Review by devwilliamy — CHANGES_REQUESTED** [review]

File: src/app/(main)/about/page.tsx (hunk 1 above)
Comment (george-icarcover) [reply src/app/(main)/about/page.tsx:25]: Its for the font family styling, I copied it from one of the other policy pages that somone made

File: src/app/(main)/contact/page.tsx [hunk 3]
```diff
@@ -1,7 +1,311 @@
...
+      errors: false,
+      errorMessage: '',
+      firstVisit: true,
+    },
+  });
+  const validateEmail = (e: ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    const isValid = emailRegex.test(value);
+    if (!isValid) {
+      console.log(value);
```
Comment (devwilliamy) [inline src/app/(main)/contact/page.tsx:75]: Get rid of console logs

**Comment by devwilliamy** [comment]
15529 should not be on the same line

image

Find out what these icons should do

image

I think the placeholder text should have a pl-1 so it's not right against the edge. Also find out how the email should work

image

**Comment by devwilliamy** [comment]
This needs to be
Coverland
15529 Blackburn Ave
Norwalk, CA 90650

image

This should be 123-456-7890

image

This needs to be optional

image

Can you also make sure it works on ur phone

**Comment by devwilliamy** [comment]
Leaving this here but I'm not too worried about it

Bad phone number doesn't stop it

image

Don't need the plus

image

Not a proper phone number

image

**Review by devwilliamy — COMMENTED** [review]
LGTM

**Pull request merged**